components/CardLocationInfo/CardLocationInfo.js

```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components";
+
+export default function CardLocationInfo({ title, children }) {
+  return (
+    <>
+      <StyledDivider />
+      <StyledH2>{title}</StyledH2>
+      <article>{children}</article>
```

Comment:
```suggestion
export default function CardLocationInfo({ title, description }) {
  return (
    <>
      <StyledDivider />
      <StyledH2>{title}</StyledH2>
      <article>{description}</article>
```
I don't see a reason to use the children prop here. For right now, as you have a hard-coded Lorem text as child it is no problem to do it like this, but in the long run you will probably want to keep the content flexible :)